docs/collecting-data/collecting-from-own-applications/out-of-the-box-tracking/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Out-of-the-box tracking"
+date: "2023-10-12"
+sidebar_position: 200
+---
+
+# Out-of-the-box tracking for Web or Mobile Events
+
+Depending on your business needs you may decide to come up with your own events / custom context entities / self-describing events. However, for out-of-the-box solutions Snowplow designed a set of events and context entities specifically useful for web and mobile tracking which we used as a base for our data models, too.
+
+In the below tables we have collected the list of such events and context entities and highlight where possible how you can enable them in the snowplow dbt packages for modeling purposes.
+
+:::tip
+Snowplow has started to move away from the wide, out-of-the-box canonical event structure and as part of this process some of these vital fields are now recommended to be tracked and used from purpose built topical context entities such as below. The snowplow-unified data model structure should enable this transition to be smooth: with a set of coalesce(..) statements you can change the source of a specific field in the `unify_fields_query()` macro, if necessary. You can check the full list of fields and from where we recommend taking them for both web and mobile [here](/docs/understanding-your-pipeline/canonical-event/index.md).
+:::
+
+## Events
+| event               | web | mobile | recommendation                                                           | how to enable it in the trackers / pipeline | how to enable it in the unified model                      | field name                                                  |
+|-----------------------|-----|--------|--------------------------------------------------------------------------|-------------------------------------------|------------------------------------------------------------|-------------------------------------------------------------|
+| screen view events [(schema)](https://github.com/snowplow/iglu-central/tree/master/schemas/com.snowplowanalytics.snowplow/screen_view/jsonschema)   | ❌   | ✅      | required for mobile (needed for screen_view_id)                          | must be manually tracked                  | snowplow__enable_mobile: true                              | com_snowplowanalytics_mobile_screen_view_1                  |
+## Context entities
+
+| context               | web | mobile | recommendation                                                           | how to enable it in the trackers / pipeline | how to enable it in the unified model                      | field name                                                  |
+|-----------------------|-----|--------|--------------------------------------------------------------------------|-------------------------------------------|------------------------------------------------------------|-------------------------------------------------------------|
```

Comment:
would it be clearer to add those "pre-" and "suffixes" in brackets maybe? it's a bit of a nightmare to update this table with all this markdown having to use tools to translate back and forth each time there is a change but I guess once solidified we could add tabs to support different warehouse outputs if otherwise doesn't look right

Reply:
Maybe just a note on the page somewhere is probably good enough